frontend/widgets/OBSBasic.cpp

```diff
@@ -42,6 +42,7 @@
 #include <utility/WhatsNewInfoThread.hpp>
 #endif
 #include <widgets/OBSProjector.hpp>
+#include <utility/ScreenshotObj.hpp>
```

Comment:
Place this below `#include <utility/SceneRenameDelegate.hpp>`.

Reply:
Fixed